app/frontend/package.json

```
@@ -82,12 +83,20 @@
     "react-test-renderer": "^16.13.1"
   },
   "jest": {
-    "collectCoverageFrom": [
-      "src/**/*.{ts,tsx}",
-      "!**/node_modules/**",
-      "!src/pb/*"
+    "preset": "ts-jest",
+    "testEnvironment": "node",
+    "transform": {
+      "node_modules/variables/.+\\.(j|t)sx?$": "ts-jest"
+    },
+    "transformIgnorePatterns": [
+      "node_modules/(?!variables/.*)"
     ]
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged && jest"
+    }
+  },
```

These lines were removed before, because in this repo prettier caused problems.
```suggestion
  "husky": {
    "hooks": {
      "pre-commit": "jest"
    }
  },
```

---

@aapeliv why/how was husky messing up the rest of the repo? Would be nice if we could keep prettify on commit, though not essential.

---

It installs hooks into the local git repo, these rules broke because it's now in a subdirectory.

I would really prefer not to use husky to install hooks into the "global" environment, is there some other way of achieving this? Could these be conditionally installed instead?

---

I think it is a good thing to have it always there. Properly configured, of course.

---

@aapeliv If the environment variable HUSKY_SKIP_INSTALL is set to 1 the hooks won't be installed when running npm install. Could that lead to a workable solution?

---

But maybe creating an npm script that runs prettier and jest is maybe the best. Then we can setup our IDE to run it on commit. Or add it to a commit hook manually.